students/11th/daseulsong/js/login.js

```diff
@@ -0,0 +1,20 @@
+'user strict';
```

Comment:
```suggestion
'use strict';
```
use strict의 오타인 듯 합니다

Reply:
오타 정정 하였습니다.